### DISCUSSION TITLE
孩子们，我发不了图片了

### DISCUSSION THREAD

**Discussion by sb35386057** [share121/inter-knot#5046] [委托]
如题目，从昨天下午开始，我发帖子评论的时候，按paste上传图片，那个键没反应，用vpn又按了几次也没有，取消vpn又按了几次也没有，有没有人知道什么鬼？

**Comment by kinich-cute** [comment]
emmmm我也是

**Comment by sb35386057** [comment]
好了孩子们我又能发了哈哈哈，委托结束
<img width="72" alt="68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f33396637636537363962613239303264626665653733353137663765656334392e706e67" src="https://github.com/user-attachments/assets/e987c272-0b1b-4c83-bb98-c877a890cd3d">

**Reply by sb35386057** [reply]
我又能发了哈

**Reply by sb35386057** [reply]
勾石一样的语音识别
